### ISSUE TITLE
The meaning of the two training maps

### ISSUE THREAD

**Issue by CH-Echo** [ultralytics/ultralytics#16266]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I would like to ask a question: why when I train Yolov8pose, there will be a series of PRmap indicators when the training reaches the epoch I specified, but after reaching the epoch, I will get another indicator. What is the difference between these two indicators?The key is that when I trained to the end, it was relatively high, but at the end there was another indicator and it dropped. I’m not sure what that means.

**Comment by CH-Echo** [comment]
Thank you for your reply. I’m not saying that the map dropped during training. I mean that after the epoch ends, an indicator will be printed. I want to know what this indicator means.

**Comment by CH-Echo** [comment]
> 👋 你好@CH-Echo，感谢您提出有关 YOLOv8 中的训练指标的问题 🚀！这是自动回复，但请放​​心，Ultralytics 工程师很快就会为您提供帮助。
> 
> 在训练 YOLOv8 时，可以同时观察 PR 图（Precision-Recall 图）和 mAP（平均准确率）指标。这里简单解释一下：
> 
> * **PR 图**：这些通常在训练期间使用，以评估模型在不同置信度水平下的精确度和召回率。它们可以洞悉不同阈值下精确度和召回率之间的权衡。
> * **mAP**：这是在一个 epoch 结束时计算的，用于全面衡量所有类别和 IoU 阈值的模型性能。它是确定整体物体检测质量的关键性能指标。
> 
> 由于学习率调整或数据集复杂度等各种因素，这些指标在不同时期之间出现波动是很常见的。接近尾声时指标下降可能表示过度拟合或需要对训练参数进行微调。
> 
> 问题提交的最佳实践：
> 
> * 请提供一个[最小可重现的示例](https://docs.ultralytics.com/help/minimum_reproducible_example/)以帮助我们了解您的设置。
> * 包括数据集样本和具体训练配置等详细信息。
> 
> 为了进一步帮助您，请考虑查看我们的[培训指南](https://docs.ultralytics.com/guides/model-training-tips/)并与社区互动：
> 
> * 加入我们的[Discord](https://ultralytics.com/discord)进行实时聊天🎧。
> * 参与[Discourse](https://community.ultralytics.com)或我们的[Subreddit](https://reddit.com/r/ultralytics)上的讨论。
> 
> 如果您要设置或排除环境故障，请确保您已`ultralytics`正确安装了该软件包：
> 
> ```shell
> pip install ultralytics
> ```
> 
> 确保您的环境满足我们的[pyproject.toml](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml)中指定的必要要求。
> 
> 我们感谢您的参与并期待解决您的疑虑！

Thank you for your reply. I’m not saying that the map dropped during training. I mean that after the epoch ends, an indicator will be printed. I want to know what this indicator means.

**Comment by CH-Echo** [comment]
Because when I evaluate, I finally get the mAP during training. I want to choose a suitable model. At present, it seems that I can only judge it based on the verification mAP data.

**Comment by Y-T-G** [comment]
What's the training code you used?

**Comment by CH-Echo** [comment]
> 您使用的训练代码是什么？

      Epoch    GPU_mem   box_loss  pose_loss  kobj_loss   cls_loss   dfl_loss  Instances       Size
    100/100      3.77G     0.2612     0.3558    0.09013      0.165     0.8059         13        640: 100%|██████████| 512/512 [02:10<00:00,  3.92it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Pose(P          R      mAP50  mAP50-95): 100%|██████████| 72/72 [00:13<00:00,  5.46it/s]
                   all       1146       2083      0.999       0.99      0.995      0.975      0.981      0.968      0.983      0.949

100 epochs completed in 4.096 hours.
Optimizer stripped from /data/ultralytics/runs/pose/train21/weights/last.pt, 53.3MB
Optimizer stripped from /data/ultralytics/runs/pose/train21/weights/best.pt, 53.3MB

Validating /data/ultralytics/runs/pose/train21/weights/best.pt...
Ultralytics YOLOv8.2.70 🚀 Python-3.8.0 torch-2.4.0+cu118 CUDA:0 (NVIDIA GeForce RTX 2060, 5924MiB)
YOLOv8m-pose summary (fused): 237 layers, 26,447,596 parameters, 0 gradients, 81.0 GFLOPs
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Pose(P          R      mAP50  mAP50-95): 100%|██████████| 72/72 [00:22<00:00,  3.19it/s]
                   all       1146       2083      0.998       0.99      0.995      0.972      0.702      0.694      0.574      0.552
Speed: 0.3ms preprocess, 12.9ms inference, 0.0ms loss, 0.8ms postprocess per image
Just like this, when the training reaches 100, it is 0.9, but there will be an indicator in the end. The following is my training code.
from ultralytics import YOLO
import torch

torch.cuda.empty_cache()

if __name__ == '__main__':
    
    # model = YOLO('/data/ultralytics/ultralytics/cfg/models/v8/yolov8-pose.yaml').load('yolov8m-pose.pt')
    # freeze_layers = 10
    # model.model.model[:freeze_layers].requires_grad_(False)
    modelpath = r'yolov8m-pose.pt'
    model = YOLO(modelpath)
    
    yamlpath = '/data/ultralytics/data/myposedata.yaml'
    
    model.train(epochs=100, 
                data=yamlpath, 
                batch=8,
                patience=50,
                save_period=10,
                augment=True,
                flipud=0.0,
                fliplr=0.3,
                weight_decay=5e-4,
                optimizer='AdamW',
                kobj=2.0,
                degrees=0.0,
                translate=0.2,
                scale=0.5,
                shear=0.0,
                perspective=0.0,
                bgr=0.0,
                mosaic=1.0,
                mixup=0.0,
                erasing=0.6,
                crop_fraction=1.0,
                lr0=0.0008,
                cos_lr=True)

**Comment by Y-T-G** [comment]
Don't add `augment=True`.

https://github.com/ultralytics/ultralytics/issues/16208#issuecomment-2342943653

**Comment by CH-Echo** [comment]
> Don't add `augment=True`.
> 
> [#16208 (comment)](https://github.com/ultralytics/ultralytics/issues/16208#issuecomment-2342943653)

why？What does this mean?

**Comment by Y-T-G** [comment]
`augment=True` isn't for training. It's used for prediction. It applies Test-Time Augmentation.

**Comment by CH-Echo** [comment]
> `augment=True`不用于训练。它用于预测。它应用测试时间增强。

Okay, I will try again, but I have a question. How do I adjust the parameters when training pose? Different parameters have different mAP. I am not particularly experienced in this area.

**Comment by Y-T-G** [comment]
For hyperparameter tuning, you can use `model.tune`

https://docs.ultralytics.com/guides/hyperparameter-tuning/

**Comment by CH-Echo** [comment]
> For hyperparameter tuning, you can use `model.tune`
> 
> https://docs.ultralytics.com/guides/hyperparameter-tuning/

Okay, let me do some research and then ask you a question: Our company is currently doing fitness exercise recognition cost data, but there is less data. What I want is the intensity of doing it on the official pre-training model. Do you have any good suggestions?

**Comment by Skillnoob** [comment]
You should get a good dataset and fine tune.
Here's a guide for tips on how datasets should look like:
https://docs.ultralytics.com/yolov5/tutorials/tips_for_best_training_results/
